src/containers/OfferProducts/OfferProducts.jsx

```diff
@@ -1,9 +1,9 @@
 import { useState, useEffect } from "react";
-import ProductCard from "../ProductCard/ProductCard";
+import ProductCard from "../../components/ProductCard/ProductCard";
 import fetchProducts from '../../services/fetchProducts';
-import styles from './OffersContainer.module.scss';
+import styles from './OfferProducts.module.scss';
 
-export default function OffersContainer() {
+export default function OfferProducts() {
```

Comment:
`ProductOffers` would be a better name.
CHat GPT is really great at making these names for you ;)